README.adoc

```diff
@@ -24,6 +24,11 @@ toc::[]
 [[img-build-status]]
 image::https://ci.centos.org/buildStatus/icon?job=vagrant-service-manager[link="https://ci.centos.org/job/vagrant-service-manager"]
 
+== Prerequisites
```

Comment:
Except the "Prerequisites" for this plugin is a Vagrant install. Whether an install of the plugin makes any sense without using a ADB/CDK VM is a different question, but the prerequisite here is Vagrant.